cluster/addons/dns/coredns/coredns.yaml.in

```diff
@@ -163,6 +165,8 @@ spec:
           readOnlyRootFilesystem: true
       dnsPolicy: Default
       volumes:
+        - name: tmp
+           emptyDir: {}
```

Comment:
Same here as well, extra indent
```
- name: tmp
  emptyDir: {}
```

Reply:
Thanks - same copy paste error in the other files too.
And there was also a tab that snuck into manifests.go.